src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
+            Ball ball = lottery.getRandomBall();
+            System.out.println(ball);
+        }
     }
```

Comment:
Consider defining the number of balls to generate as a constant at the class level. This makes it easier to change the value in one place and enhances code readability by giving the magic number a meaningful name.